qa/workunits/rbd/krbd_fallocate.sh

```diff
@@ -107,7 +104,7 @@ assert_deallocated
 # unaligned blkdev_issue_discard
 allocate
 py_blkdiscard $((OBJECT_SIZE / 2))
-assert_deallocated_unaligned 1
+assert_zeroes_unaligned $NUM_OBJECTS
```

Comment:
Nit: you would need to cherrypick a7d89c07946f89ad28987d65d9ba71c464c32431 if you wanted to use the new function name. Otherwise, you need to keep it ```assert_deallocated_unaligned```